test/others/criu-ns/run.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
```

Comment:
@warusadura Could you add this file in the [`lint`](https://github.com/checkpoint-restore/criu/blob/33dd66c6fc93c47213aaa0447a94d97ba1fa56ba/Makefile#L433) makefile target?

Reply:
will do, thanks!

Reply:
I could access the `name` property of `file` as in,
`file.name`